examples/lib/stories/games/padracing/background.dart

```diff
@@ -0,0 +1,25 @@
+import 'dart:ui';
+
+import 'package:examples/stories/games/padracing/game.dart';
+import 'package:flame/components.dart';
+import 'package:flame/extensions.dart';
+import 'package:flame/palette.dart';
+import 'package:flutter/material.dart' hide Image, Gradient;
+
+class Background extends PositionComponent
+    with HasGameRef<PadRacingGame>, HasPaint {
+  Background() : super(size: PadRacingGame.trackSize, priority: 0);
```

Comment:
priority 0 is redundant right?

Reply:
This whole component is redundant, it only paints a black rectangle :sweat_smile: 
(It used to do cooler things...)